lms/djangoapps/email_marketing/migrations/0011_unsubscribeduser.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
@Fahminajjar please create the migration (and its related models) in another Django app to make future release migrations smoother.

I recommend this reading to explain the issue: https://riptutorial.com/django/example/23659/solving-migration-conflicts

A quick hack that I found useful, was to use the `UserProfile.meta` field and stuff things in it. Creating a django model in an Edraak-owned app e.g. `edraak_misc` is a good solutions as well.